website/docusaurus.config.js

```diff
@@ -266,7 +266,9 @@ var siteSettings = {
     "/css/fonts.css",
     "/css/entypo.css",
     "/css/search.css",
-    "/css/api.css",
+    "/css/redoc.css",
+    "/css/stoplight-base.css",
+    "/css/stoplight-custom.css",
```

Comment:
Hey @GideonShils, when the css files are added within the docusaurus config, it will add them to every page throughout the site. For example, I see styles from `stoplight-base.css` being applied on the homepage.

Most are overridden by custom styles, but it might be difficult to promise styles on all pages are not affected. 

Rather than adding them here in the config, would it be possible to add the css files directly in the `src/pages/dbt-cloud/api*` files to make them only available on those pages?

<img width="466" alt="image" src="https://github.com/dbt-labs/docs.getdbt.com/assets/13647110/a4e4ca8e-d919-4dc8-8a19-aac2773cac11">

Reply:
I ended up removing the css files from the docusaurus config and imported them directly in the stoplight component. I kept them as separate stylesheets as opposed to CSS modules so that they would only load on the API docs pages and not on the rest of the site. Tested to ensure:
- New styles do not load on rest of site: headings look correct, bullet points appear as expected
- Styles load on new pages: nav bar has correct font, content has correct custom styles